src/templates/masthead.html

```diff
@@ -0,0 +1,12 @@
+<div class="usa-navbar">
+  <button class="usa-menu-btn">Menu</button>
+  <div class="usa-logo" id="logo">
```

Comment:
Is there reason for `logo` to be split into own component, or is that not necessary?